pyodata/v2/model.py

```diff
@@ -1933,7 +1939,10 @@ def from_etree(entity_type_property_node):
             sap_attribute_get_string(entity_type_property_node, 'text'),
             sap_attribute_get_bool(entity_type_property_node, 'visible', True),
             sap_attribute_get_string(entity_type_property_node, 'display-format'),
-            sap_attribute_get_string(entity_type_property_node, 'value-list'), )
+            sap_attribute_get_string(entity_type_property_node, 'value-list'),
+            # End of TODO, regular attributes below
```

Comment:
The TODO comment is probably a leftover

Reply:
Evidently not a great comment, but it was meant to clearly separate the coming line from the sap_attribute_get... lines above, which have been commented as "TODO, regular attributes below". Will think of something better.